src/topics/resolver/github-resolver.ts

```diff
@@ -22,37 +29,69 @@ export class GithubResolver implements IResolver {
     };
   }
 
-  resolve = async (githubData: string): Promise<string> => {
-    const splitGithubData = githubData.split(":");
-    if (splitGithubData.length === 3) {
-      const id = githubData.split(":")[2];
-      const resolvedAccount = resolveAccount("1001", id);
-      return resolvedAccount;
-    }
+  resolve = async (githubData: FetchedData): Promise<FetchedData> => {
+    let githubDataUpdated = Object.entries(githubData).filter(
+      ([account, value]) => {
+        const splitGithubData = account.split(":");
+        if (splitGithubData.length === 3) {
+          const id = account.split(":")[2];
+          this.resolvedAccounts[resolveAccount("1001", id)] = value;
+        }
+        return splitGithubData.length !== 3;
+      }
+    );
+
+    githubDataUpdated = githubDataUpdated.map((data) => {
+      return [data[0].split(":")[1], data[1]];
+    });
+
+    const githubAccounts = githubDataUpdated.map((item) => item[0]);
+
+    const resolveGithubHandles = async (username: string[]): Promise<void> => {
```

Comment:
this function should be another function of the github resolver

Reply:
don't declare its implementation in the resolve method

Reply:
I saw we did this for the old big resolver function before (that's why I did this too). Is it just cleaner to put the function out of the other or are there any other reasons?